raysort/sort_utils.py

```diff
@@ -195,8 +200,8 @@ def generate_part(
             )
         elif cfg.azure_containers:
             azure_utils.upload(path, pinfo)
-        logging.info("Generated input %s", pinfo)
-        return pinfo
+        logging.info("Generated input %s, checksum: %s", pinfo, stderr)
+        return pinfo, stderr.strip("\n")
```

Comment:
Let's actually add a field to PartInfo, and stick this checksum in it.

Reply:
I think @kennethlien was worried about the additional write to S3. Is that a negligible concern?

Reply:
No this won't incur any additional writes to S3. The PartInfo file is saved locally on the head node. See `input-manifest-cloud.csv` in your local directory.